client/my-sites/checkout/upsell-nudge/difm-upsell/index.jsx

```diff
@@ -140,8 +146,8 @@ export class DifmUpsell extends PureComponent {
 						</p>
 
 						<p>
-							{ translate( '{{b}}Custom websites starting at $4,900{{/b}}', {
-								components: { b: <b /> },
+							{ translate( '{{em}}Custom websites starting at $4,900.{{/em}}', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.